extensions/pyRevitDevHooks.extension/hooks/doc-updater.py

```diff
@@ -5,4 +5,4 @@
 
 for wall in revit.query.get_elements_by_class(DB.Wall, doc=doc):
     p = wall.LookupParameter("Unconnected Height")
-    p.Set(5)
+    print("wall: {} Unconnected Height: {}".format(wall.Id.ToString(), p.AsDouble()))
```

Comment:
Don't use `ToString` in python code. I recommend write `str(wall.Id)`

Reply:
fixed